### PR TITLE
DBP-668 Increase fixup-rocketchat memory limit to 200Mi

### PR DESCRIPTION
**Pull request by JannikBadenhop**
# Description
Increase fixup-rocketchat memory limit from 100Mi to 200Mi because it is crashing on every execution of mongosh since update to k8s version 1.28.7

## Links to Tickets or other pull requests
[DBP-668](https://ticketsystem.dbildungscloud.de/browse/DBP-668)

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by dimapin — APPROVED** [review]

File: ansible/group_vars/production/resources.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ ROCKETCHAT_MEMORY_LIMITS: "4Gi"
 ROCKETCHAT_MEMORY_REQUESTS: "4Gi"
 ROCKETCHAT_FIXUP_CPU_LIMITS: "1000m"
 ROCKETCHAT_FIXUP_CPU_REQUESTS: "100m"
-ROCKETCHAT_FIXUP_MEMORY_LIMITS: "100Mi"
+ROCKETCHAT_FIXUP_MEMORY_LIMITS: "200Mi"
 ROCKETCHAT_FIXUP_MEMORY_REQUESTS: "100Mi"
```
Comment (mamutmk5) [inline ansible/group_vars/production/resources.yml:77]: Then fix up also the memory requests to the same vaule

File: ansible/group_vars/reference/resources.yml [hunk 2]
```diff
@@ -73,7 +73,7 @@ ROCKETCHAT_MEMORY_LIMITS: "4Gi"
 ROCKETCHAT_MEMORY_REQUESTS: "4Gi"
 ROCKETCHAT_FIXUP_CPU_LIMITS: "1000m"
 ROCKETCHAT_FIXUP_CPU_REQUESTS: "100m"
-ROCKETCHAT_FIXUP_MEMORY_LIMITS: "100Mi"
+ROCKETCHAT_FIXUP_MEMORY_LIMITS: "200Mi"
 ROCKETCHAT_FIXUP_MEMORY_REQUESTS: "100Mi"
```
Comment (mamutmk5) [inline ansible/group_vars/reference/resources.yml:77]: Then fix up also the memory requests to the same vaule

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

**Comment by mamutmk5** [comment]
Add the Ticket nummber to the PR titel

File: ansible/roles/rocketchat/templates/fixup-job.yml.j2 [hunk 3]
```diff
@@ -36,10 +36,10 @@ spec:
             resources:
               limits:
                 cpu: {{ ROCKETCHAT_FIXUP_CPU_LIMITS|default("1000m", true) }}
-                memory: {{ ROCKETCHAT_FIXUP_MEMORY_LIMITS|default("100Mi", true) }}
+                memory: {{ ROCKETCHAT_FIXUP_MEMORY_LIMITS|default("200Mi", true) }}
               requests:
                 cpu: {{ ROCKETCHAT_FIXUP_CPU_REQUESTS|default("100m", true) }}
-                memory: {{ ROCKETCHAT_FIXUP_MEMORY_REQUESTS|default("50Mi", true) }}
+                memory: {{ ROCKETCHAT_FIXUP_MEMORY_REQUESTS|default("200Mi", true) }}
```
Comment (mamutmk5) [inline ansible/roles/rocketchat/templates/fixup-job.yml.j2:42]: no, only for prod and ref request = limit for Memory

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

**Comment by mamutmk5** [comment]
also put the link to the ticket in the pr

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**